src/components/button/button.tsx

```diff
@@ -66,8 +67,15 @@ export class Button {
     @State()
     private justLoaded = false;
 
+    @Element()
+    private host: HTMLElement;
+
     private justLoadedTimeout?: number;
 
+    public componentWillLoad() {
+        makeClickable(this.host);
```

Comment:
Hm come to think of it, this is a bad name 😅 
I'll probably change it to something like `makeEnterClickable`